apps/parsley/src/components/LogRow/RowRenderer/index.tsx

```diff
@@ -87,7 +88,16 @@ const ParsleyRow: RowRendererFunction = ({ processedLogLines }) => {
     }
 
     if (isSubsectionHeaderRow(processedLogLine)) {
-      return <div>subsection start</div>;
+      return (
+        <SubsectionHeader
```

Comment:
What do you think of adding the command number to the header itself? E.g. something like "(1 of 3)" for this command:
<img width="1088" alt="image" src="https://github.com/evergreen-ci/ui/assets/9298431/e248ad7b-3765-4d16-b0c4-6a483dbce80e">

Reply:
It's a good idea. I just opened this ticket: https://jira.mongodb.org/browse/DEVPROD-8683